taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb

```diff
@@ -0,0 +1,2622 @@
+{
```

Comment:
<p>Same question - I think this has effectively just renamed the column <code>library_preparation_has_output</code> to <code>processed_sample_id3</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Yep!  Will do. Good catch.  That simplifies things significantly.</p>